src/components/AddBookForm.js

```diff
@@ -38,12 +43,14 @@ export default function AddBookForm({
   function handleSubmit(event) {
     event.preventDefault()
     const form = event.target
-    const { title, authors, readingSince, onPage } = form.elements
+    const { title, authors, readingSince, onPage, finishedOn } = form.elements
     onHandleCreateNewBook({
       title: title.value,
       authors: authors.value,
-      readingSince: readingSince.value ? readingSince.value : getToday(),
-      onPage: onPage.value,
+      finished: isFinished,
+      readingSince: !isFinished ? readingSince.value : '',
+      finishedOn: isFinished ? finishedOn.value : '',
+      onPage: !isFinished ? onPage.value : '',
       thumbnail: !searchedBook ? bookcover : searchedBook.thumbnail,
```

Comment:
```suggestion
      readingSince: isFinished ? '' : readingSince.value,
      finishedOn: isFinished ? finishedOn.value : '',
      onPage: isFinished ? '' : onPage.value,
      thumbnail: searchedBook ? searchedBook.thumbnail : bookcover ,
```

It's **really** minor. You basically want to check `(isFinished !== true)`. What you write is `(!isFinished === true)`. On the second one, in the background you basically add 1 more check, because the `!isFinished` is checking for 
`((isFinished === false) === true)`

This really feels so minor. The difference here is that you know that something will happen on either decision. It does not matter if it is true or false, you want to give back something. On a normal if check I would see it differently.

Okay, the more I write, the less I think it becomes important. On a tenary I would just simply go for not checking for `!isFinished` and `isFinished` instead.

Reply:
Haven't thought about this yet! But makes totally sense to me. Will change it. :-) Thank you for your explanation!